avm/res/storage/storage-account/main.bicep

```diff
@@ -188,9 +188,6 @@ param keyType string?
 @description('Optional. Key vault reference and secret settings for the module\'s secrets export.')
 param secretsExportConfiguration secretsExportConfigurationType?
 
-@description('Optional. Enables Advanced Threat Protection on the storage account.')
-param enableAdvancedThreatProtection bool = true
```

Comment:
Hey @thecsw,
thank you for your PR. I take it you need the feature for your use case though. So how about not removing it, but instead just setting the default to `false` and changing the description to point out what it says in the error message? I.e., the whole thing about being somewhat deprecated and stuff?